translations/japanese/core-commands.ts

```diff
@@ -0,0 +1,132 @@
+		"/mee - must not start with a letter or number": "文字や数字を最初においてはいけません。",
+		"What?! How are you not more excited to battle?! Try /battle! to show me you're ready.": "バトルが盛り上がってないって!?覚悟を決めて、/battle だ",
+		"Access denied for custom avatar - make sure you're on the right account?": "このカスタムアバターを使用する権限がありません。許可されているアカウントでログインしてください。",
+		"Invalid avatar.": "アバター名が正しくありません。",
+		"Avatar changed to:": "アバターを次のものに変更しました。",
+		"Artist: ": "作者",
+		"No one has PMed you yet.": "誰もあなたにPMを送っていません。",
+		"You forgot the comma.": "コンマを忘れていませんか?",
+		"User ${targetUsername} not found. Did you misspell their name?": "ユーザー名:${targetUsername}は見つかりませんでした。名前を間違えていませんか?",
+		"User ${targetUsername} is offline.": "ユーザー名:${targetUsername}は見つかりませんでした。",
+		"The user \"${targetUsername}\" was not found.": "ユーザー名:\"${targetUsername}\"は見つかりませんでした。",
+		"The room \"${target}\" was not found.": "チャットルーム:\"${target}\"は見つかりませんでした。",
+		"You do not have permission to invite people into this room.": "あなたはこの部屋でユーザーを招待する権限を持っていません。",
+		"This user is already in \"${targetRoom.title}\".": "このユーザーはすでに\"${targetRoom.title}\"にいます。",
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "/busyでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"Setting status messages in /away is no longer supported. Set a status using /status.": "/awayでのステータスメッセージの設定はサポートされなくなりました。ステータスメッセージの設定は、/statusを使って行ってください。",
+		"User '${target}' not found.": "ユーザー名'${target}'は見つかりませんでした。",
+		"${targetUser.name} does not have a status set.": "${targetUser.name}はステータスメッセージを設定していません。",
+		"${targetUser.name}'s status \"${targetUser.userMessage}\" was cleared by ${user.name}${displayReason}": "${targetUser.name}のステータスメッセージは${user.name}によって消去されました。${displayReason}",
+		"You don't have a status message set.": "ステータスメッセージを設定していません。",
+		"You have cleared your status message.": "ステータスメッセージを消去しました。",
+		"This user has not played any ladder games yet.": "このユーザーはladderが有効なバトルをしていません。",
+		"W[TN: initial for Wins]": "'Wins'の頭文字",
+		"L[TN: initial for Losses]": "'Losses'の頭文字",
+		"You already have the temporary symbol '${group}'.": "あなたはすでに一時的なランクである'${group}'になっています。",
+		"You must specify a valid group symbol.": "有効なランクの記号を特定する必要があります。",
+		"You may only set a temporary symbol below your current rank.": "現在のランクより下の一時的なシンボルしか設定できません。",
+		"Your temporary group symbol is now": "あなたの一時的なランクの記号",
+		"Currently, you're viewing Pokémon Showdown in ${language}.": "現在、Pokémon Showdownを${languege}で表示しています。",
+		"Valid languages are: ${languages}": "有効な言語一覧: ${langueges}",
+		"Pokémon Showdown will now be displayed in ${language} (except in language rooms).": "Pokémon Showdownは言語部屋以外の場所は${languege}で表示されます。",
+		"Note that rooms can set their own language, which will override this setting.": "言語部屋ではそれぞれが独自の言語を設定でき、その場合はユーザー設定より部屋の設定が優先されます。",
+		"/updatesettings expects JSON encoded object.": "/updatesettings を使用するには、JSONに変換されたファイルが必要です。",
```

Comment:
```suggestion
		"/updatesettings expects JSON encoded object.": "/updatesettings を使用するには、JSONにエンコードされたデータが必要です。",
```

使い方の例：/updatesettings {"blockChallenges": false}

このコマンドの引数は添付ファイルなどではありませんので多少不正確な説明かと思っております。
This command doesn't reference a file, so I don't think this is an accurate explanation.

Reply:
Is array allowed?
example:
`/updatesettings [{"blockChallenges": true}, {"blockPMs": true}]`

Reply:
> Is array allowed? example: `/updatesettings [{"blockChallenges": true}, {"blockPMs": true}]`

No, it's just a JSON object - ie `/updatesettings {"blockPMs": true}`.

Reply:
ok ty